src/map.h

```diff
@@ -1209,6 +1210,15 @@ void add_corpse( const tripoint &p );
     void build_floor_cache( int zlev );
     // We want this visible in `game`, because we want it built earlier in the turn than the rest
     void build_floor_caches();
+
+    /** Get random tile on circumference of a circle */
+    tripoint random_perimeter( const tripoint& src, int radius ) const
```

Comment:
Yes, a separate PR could consider yanking them out (possibly as a global function in `line.h`)? I'm wary of making too many infrastructure changes in one PR.